packages/datadog-instrumentations/src/protobufjs.js

```diff
@@ -0,0 +1,135 @@
+const shimmer = require('../../datadog-shimmer')
+const { addHook, AsyncResource } = require('./helpers/instrument')
+
+const tracingChannel = require('dc-polyfill').tracingChannel
+const serializeChannel = tracingChannel('apm:protobufjs:serialize')
```

Comment:
This will create all channels for both serialize and deserialize, where serializeChannel and deserializeChannel will both return start, end, asyncStart, asyncEnd and error channels. Was this your intent since later you wrap one channel, though what will be wrapped is a collection of channels....

you can do if you mean to wrap just the start channel... 
```
wrapOperation(messageClass, 'encode', serializeChannel.start)
```

Reply:
Ahhhh, I did not know that code will return all channels, yes ideally lets just stick to wrapping the specific channels. For serialize, I will switch the code to only use the start channel, and same for deserialize with the end channel. Thanks!

Reply:
Fixed this.

Reply:
I reverted this change as the underlying start, end channels are untraced and do not provide tracing helper functionality.